tensorflow_serving/tools/pip_package/build_pip_package.sh

```diff
@@ -44,9 +46,20 @@ function main() {
 
   cp bazel-serving/tensorflow_serving/apis/*_grpc.py \
     "${TMPDIR}/tensorflow_serving/apis"
+  
```

Comment:
Can you remove this extra space?

Reply:
Sorry, it's my mistake. I'll revert back it.

Reply:
@gautamvasudevan Sorry, it's my mistake. I'll revert back it.

Reply:
@gautamvasudevan Done. Is it okay?

Reply:
Hmm, I don't see a new commit with this change. Just the original 2.